examples/Tutorials/Basic Button/JustRideTests/JustRideTests.swift

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
If these tutorials don't have tests, we can probably remove these test files & test target